airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/AirbyteConnectorRunner.kt

```diff
@@ -17,6 +17,13 @@ import picocli.CommandLine.Model.ArgGroupSpec
 import picocli.CommandLine.Model.OptionSpec
 import picocli.CommandLine.Model.UsageMessageSpec
 
+// A micronaut environment, so that connectors can override micronaut properties
+// without needing to copy an entire application.yaml out of the core CDK.
+// Intended usage is for connectors to create
+// `destination-foo/src/main/resources/application-override.yaml` with the
+// specific properties to override.
+const val OVERRIDE_ENV = "override"
```

Comment:
Nit: override sounds like it's non-standard behavior, but is really just the connector level configuration. Consider "connector" or similar as the env name imho.

Reply:
done in https://github.com/airbytehq/airbyte/pull/51600/commits/fa1d8075e0a8b54fb49c04b2afacf815165c369c